src/core/ruby_version.rs

```diff
@@ -2658,8 +3000,8 @@ mod tests {
     #[test]
     fn test_get_ruby_stack_trace_3_3_1() {
         let source = coredump_3_3_0();
-        let vm_addr = 0x7f7ff21f1868;
```

Comment:
These and others needed to be updated since I replaced the coredump with a non-truncated on.